### DISCUSSION TITLE
Elfogadó nyugta - megjegyzés

### DISCUSSION THREAD

**Discussion by kdenes6500** [nav-gov-hu/M2M#35] [Kérdések] [answered]
Ha az XML-ben a <megjegyzes> részbe szöveget teszünk (pl. 'T1041 esetén az érintett személy neve), akkor az ÁNYK-ból beküldött nyomtatvány esetén ez az információ rákerül az elfogadó nyugtára.
M2M-beküldés esetén ez az információ hiányzik az elfogadó nyugtáról.

**Comment by TamasPill** [comment, marked as answer]
Megtaláltuk az okát, hogy ez miért nem kerül jelenleg átadásra.
Egy december eleji release-be betesszük a javítást.
